src/Jackett.Common/Definitions/hdforever.yml

```
@@ -292,13 +270,14 @@ search:
           args: [".VFQ.MULTI", ".MULTI.VFQ"]
         - name: append
           args: "{{ if .Result._release_group }}-{{ .Result._release_group }}{{ else }}{{ end }}"
-    title_vostfr:
+
+    title_VOSTFR:
```

```suggestion
    title_vostfr:
```

Again, just keep these as lowercase.

---

revert done